### PR TITLE
Rec: Prepare for rec-4.3.0-beta1

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by omoerbeek** [comment]
https://powerdnsblog.wordpress.com/?p=2982&preview=true

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 1]
```diff
@@ -1,6 +1,85 @@
...
+    :pullreq: 8560
+
+    Check return value of dup() and avoid fd leak if if fdopen() fails
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 8559
+    :tickets: 8558
+
+    Avoid startup race by setting the state of a tread before starting it.
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.3.rst]:
```suggestion
    Avoid startup race by setting the state of a thread before starting it.
```

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 2]
```diff
@@ -1,6 +1,85 @@
...
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8561
+
+    QName Minimization is no longer experimental
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.3.rst]: Does this mean it now defaults to on?

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 3]
```diff
@@ -1,6 +1,85 @@
...
+    :tags: Improvements
+    :pullreq: 8440
+
+    Fix -WShadow warnings (Aki Tuomi)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8477
+
+    Enable qname minimization
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.3.rst]: Maybe combine 8477 and 8561 into one changelog item.

File: docs/secpoll.zone [hunk 4]
```diff
@@ -205,6 +205,7 @@ recursor-4.2.1.security-status                          60 IN TXT "1 OK"
 recursor-4.3.0-alpha1.security-status                   60 IN TXT "1 OK"
```
Comment (Habbie) [inline docs/secpoll.zone]: Maybe mark these 3 as unsupported?

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 5]
```diff
@@ -1,6 +1,79 @@
...
+    :pullreq: 8559
+    :tickets: 8558
+
+    Avoid startup race by setting the state of a thread before starting it.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8561, 8477
+
+    QName Minimization is no longer experimental and enabled by default.
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.3.rst]:
```suggestion
    QName Minimization is no longer experimental and is now enabled by default.
```

**Pull request merged**